src/constants/pools.ts

```diff
@@ -34,7 +36,8 @@ export const POOLS = {
       '0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014',
       '0xf4c0dd9b82da36c07605df83c8a416f11724d88b000200000000000000000026',
       '0xf8a0623ab66f985effc1c69d05f1af4badb01b0000020000000000000000001d',
-      '0x021c343c6180f03ce9e48fae3ff432309b9af19900020000000000000000000b'
+      '0x021c343c6180f03ce9e48fae3ff432309b9af19900020000000000000000000b',
+      '0x87165b659ba7746907a48763063efa3b323c2b0700020000000000000000002d'
```

Comment:
This list is meant to contain only pools that are actively managed by Gauntlet

Reply:
Am I wrong that the current state is it manages all of them with that owner, but the new Authorizer will make it fine-grained?

Reply:
They can manage all but will only manage the ones in the original list

Reply:
Ok then; restoring original list